bc_obps/registration/schema/validators.py

```diff
@@ -1,10 +1,10 @@
 from typing import Optional
 
 
-def validate_cra_business_number(value: int) -> Optional[int]:
+def validate_cra_business_number(value: str) -> Optional[str]:
     """
     Validate that the cra_business_number has 9 digits.
     """
-    if not (isinstance(value, int) and 100000000 <= value <= 999999999):
+    if not (value.isdigit() and len(value) == 9):
```

Comment:
I would use a regex here, the `isdigit()` method can return true with funny characters https://stackoverflow.com/a/54912545